server/models/Professional.ts

```diff
@@ -0,0 +1,51 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import {User} from './User';
+
+export enum Race {
+    WHITE = "white",
+    BLACK = "black",
+    ASIAN = "asian",
+    LATINX = "latinx",
+    INDIGENOUS = "indigenous",
+    DECLINE = "decline",
+    GHOST ="ghost"
+}
+
+export enum Education {
+    ASSOCIATES = "associates",
+    BACHELORS = "bachelors",
+    MASTERS = "masters",
+    PHD = "doctorate",
+    CERT = "industry certification",
+    GHOST = "ghost"
+}
+
+
+@Entity()
+export class Professional extends User {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column({ length: 100, nullable: false })
+  name!: string;
+
+  @Column({ nullable: false })
+  password!: string;
+
+  @Column({ nullable: false, length: 512 })
+  email!: string;
```

Comment:
Note, this seems to violate DRY since these fields are already defined in the User class.  Is there some way we can dedupe these?  Do these fields need to be defined here if they're already defined in the User abstract class?